Formula/opencsg.rb

```diff
@@ -0,0 +1,74 @@
+class Opencsg < Formula
+  desc "The CSG rendering library"
+  homepage "http://www.opencsg.org"
+  url "http://www.opencsg.org/OpenCSG-1.4.0.tar.gz"
+  sha256 "ecb46be54cfb8a338d2a9b62dec90ec8da6c769078c076f58147d4a6ba1c878d"
+
+  depends_on "qt5" => :build
+  depends_on "glew"
+
+  def install
+    system "qmake", "-r", "INSTALLDIR=#{prefix}",
+      "INCLUDEPATH+=#{HOMEBREW_PREFIX}/include",
+      "LIBS+=-L#{HOMEBREW_PREFIX}/lib -lGLEW"
+
+    system "make", "install"
+  end
+
+  def patches
```

Comment:
We haven't used `def patches` syntax for a _long_ time, heh. You'll want to submit the patch to https://github.com/Homebrew/formula-patches & then use a `patch do` blob. Has this patch been submitted upstream though?